src/layer/__tests__/cache.ts

```diff
@@ -432,3 +486,62 @@ describe('Unit test for cacheStillValid', () => {
     expect(isValid).toBeFalsy();
   });
 });
+
+describe('test getCapabilities', () => {
```

Comment:
Do we need these tests? Unless I'm missing something, I think their functionality (with regards to caching) is replicated elsewhere... or is the difference in that we are caching GET requests instead of POST?

Also, note that `fetchGetCapabilitiesJson` is not a public method (not sure if it matters in this case).

Reply:
These tests are not needed, but they are the only tests that are testing cache and using GET requests. I would say there are better methods to test GET requests and caching, so I will remove these tests